src/functionalTest/java/uk/gov/hmcts/reform/prl/controllers/managedocuments/ManageDocumentsControllerFunctionalTest.java

```diff
@@ -64,6 +67,21 @@ public void givenManageDocuments_whenCopy_manage_docsEndPoint_thenRespWithCopied
             .assertThat().statusCode(200);
     }
 
+    @Test
+    public void givenManageDocuments_GiveErrorWhenCourtAdminUserSelectCourt() throws Exception {
```

Comment:
please add multiple test cases - both positive and negative scenarios.

Reply:
@yogendraU  as discussed add configuration for judge to validate the positive scenario